RecoParticleFlow/PFClusterProducer/src/SpikeAndDoubleSpikeCleaner.cc

```diff
@@ -117,10 +117,55 @@ clean(const edm::Handle<reco::PFRecHitCollection>& input,
     }    
     const spike_cleaning& clean = _thresholds.find(hitlayer)->second;    
     if( rechit.energy() < clean._singleSpikeThresh ) continue;
+
+
+
+    //Fix needed for HF
+    float compsumE = 0.0;
+    if ((hitlayer==12 or hitlayer==11))
+	{
+	int predphi =2;
+
+	int comp = std::abs(hitlayer-13);
+	const HcalDetId& detid = (HcalDetId)rechit.detId();
+	std::vector<uint32_t> RawDetIds;
+	int heta = detid.ieta();
+	int hphi = detid.iphi();
+	
+	if (std::abs(heta)>39) predphi=4;
+
+ 	int curphiL = hphi-predphi;
+	int curphiH = hphi+predphi;
+
+	if (predphi>abs(hphi) or predphi>abs(72-hphi))
```

Comment:
My frame of reference for this range (and most of these methods) is here 

https://github.com/cms-sw/cmssw/blob/CMSSW_9_1_X/RecoLocalCalo/HcalRecAlgos/src/HcalHF_S9S1algorithm.cc

Here, the "magic" numbers 39,4,72 are not from an exterior definition.  I will look further however